metabase/metabase
# Add new environment setting reset-token-ttl-hours

Closes https://github.com/metabase/metabase/issues/12829

### Description

Add new environment setting `reset-token-ttl-hours`. No setter has been defined to limit env values.

### How to verify

Modify the `MB_RESET_TOKEN_TTL_HOURS`. verify in REPL or `GET "/password_reset_token_valid"`

1. Set  `MB_RESET_TOKEN_TTL_HOURS` value to 1 in shell. Startup Metabase
2. In REPL inspect `(env/env :mb-reset-token-ttl-hours)`. Verify value is 1
3. Wait 1 hour
4. Make a call to `/password_reset_token_valid`. Verify value is false

### Demo
NA

### Checklist

- [X ] Tests have been added/updated to cover changes in this PR.
